src/shared.js

```diff
@@ -2015,4 +2015,18 @@ export const translateNumToType = {
   "0052": "NA",
   "0053": "NA",
   "0054": "NA"
-};
\ No newline at end of file
+};
+
+export const getCollectionsByVisitType = async (connectId, visitType) => {
```

Comment:
There should already be a function you can use for getting collections by visit type

Reply:
resolved
https://github.com/episphere/biospecimen/pull/294/commits/981ccf6773cb6fa715996a70cbc012b9a7ba035e